src/jagdish/A21_Assignemnt.java

```diff
@@ -0,0 +1,24 @@
+package jagdish;
+
+public class A21_Assignemnt {
+	String temp = "";
```

Comment:
if this variable is common for other methods then we should write it at the instance level.